### ISSUE TITLE
Tab trap in JupyterLab notebook code cell

### ISSUE THREAD

**Issue by gabalafou** [Quansight-Labs/jupyter-a11y-mgmt#168]
### Description

When tabbing through the JupyterLab UI with a notebook open, the keyboard gets trapped in a notebook cell.

I'm not sure if this is technically a violation of [WCAG 2.1.2 - No Keyboard Trap](https://www.w3.org/TR/WCAG22/#no-keyboard-trap) because the user can use the up/down arrow keys to move from cell to cell. However, once the user reaches the last cell, they cannot go any further. However, the only way to tab to parts of the UI that are subsequent to the last code cell, would be to tab backwards (<kbd>shift</kbd> + <kbd>tab</kbd>) through the UI. This is a bad user experience.

### Value / benefit

Keyboard traps are a major accessibility issue, explained in [Understanding Success Criterion 2.1.2: No Keyboard Trap](https://www.w3.org/WAI/WCAG22/Understanding/no-keyboard-trap.html).

### Implementation details

I'm not entirely sure how to fix this issue. But my thinking is that a first attempt could be a PR that changes the UX so that the tab key does not drop the user into the cell. In order to enter a code cell, the user has to press enter. 

### Tasks to complete

- [ ] Confer with team about proposed implementation (i.e., pressing enter key to begin editing notebook cell)
- [ ] Submit PR to remove code cell tab trap
